pkg/apis/apps/group.go

```diff
@@ -97,10 +97,10 @@ const (
 	APPNAME               CliOption = "appname"
 	DATA                  CliOption = "Data"
 	ZONE                  CliOption = "zone"
-	USE_BASIC_AUTH        CliOption = "use_basic_auth"
-	USE_ISTIO             CliOption = "use_istio"
-	DELETE_STORAGE        CliOption = "delete_storage"
-	DISABLE_USAGE_REPORT  CliOption = "disable_usage_report"
+	USE_BASIC_AUTH        CliOption = "use-basic-auth"
```

Comment:
Could we remove the options USE_BASIC_AUTH and USE_ISTIO? I don't think these options are used anymore. I think those options predated the move to KFDef.

Reply:
@jlewi Done!